test/e2e/specs/specs-calypso/wp-manage-domains-spec.js

```diff
@@ -187,16 +187,16 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function
 			return await enterADomainComponent.enterADomain( blogName );
 		} );
 
-		it( 'Can add domain to the cart', async function () {
+		it.skip( 'Can add domain to the cart', async function () {
 			const enterADomainComponent = await EnterADomainComponent.Expect( driver );
 			return await enterADomainComponent.clickonAddButtonToAddDomainToTheCart();
 		} );
 
-		it( 'Can see checkout page', async function () {
+		it.skip( 'Can see checkout page', async function () {
 			return await MapADomainCheckoutPage.Expect( driver );
 		} );
 
-		it( 'Empty the cart', async function () {
+		it.skip( 'Empty the cart', async function () {
```

Comment:
FYI, not emptying the cart caused an issue where the cart started timing out due to too many items to be loaded. See the failure caused by this in [this build](https://teamcity.a8c.com/viewLog.html?buildId=6225523&buildTypeId=calypso_RunCalypsoE2eDesktopTests). I've emptied the cart manually and will bring back the emptying spec in the `after` hook here. cc @scinos @hambai